core/src/main/java/hudson/triggers/TriggerAdminMonitor.java

```diff
@@ -0,0 +1,79 @@
+package hudson.triggers;
+
+import hudson.ExtensionList;
+import hudson.model.AdministrativeMonitor;
+import hudson.util.CopyOnWriteMap;
+import jenkins.model.Jenkins;
+import org.kohsuke.stapler.HttpResponse;
+import org.kohsuke.stapler.HttpResponses;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Logger;
+
+public class TriggerAdminMonitor extends AdministrativeMonitor {
```

Comment:
This should be better restricted unless we want clients to consume this. Also `SlowTriggerAdminMonitor` feels more explicit.